themes/edx.org/lms/static/sass/partials/base/_certificates.scss

```diff
@@ -0,0 +1,27 @@
+// Certificate overrides for edx.org
+
+.certificate-card {
+    &.mode-honor, &.mode-audit {
+        border-color: $honor-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/honor.png');
```

Comment:
Do we need an honor cert example on the sandbox? 
Also, as discussed, do we need PDF examples or do we assume that the cert service just works?